model/asset.go

```diff
@@ -1,9 +1,10 @@
-package conversion
+package model
```

Comment:
I am not good at naming, what do u suggest?

Reply:
I'm not sure I necessarily have a better suggestion; I'm used to `model` for database models, so it seemed a bit odd. Ideally I'd prefer to match some kind of convention if there is one.

Reply:
name it `common`? if it's called asset, then golang warns it stutters like asset.Asset.

Reply:
After reading through the go style guide on this & linked docs.... I think that `models` and `common` are both pretty generic and it would probably be better to find a more specific name.

As far as `asset.Asset` - I think that would be better, and it would match the pattern of [`ring.Ring`](https://go.dev/doc/effective_go#package-names) from Effective Go... but you said go complains about it because of the duplication, I think?

If we had `asset` as the package name, we could do things like `asset.FromREST`, `asset.New`, or similar, which feels idiomatic.

But if that doesn't work - what about `caiasset`? If that doesn't work, maybe just `cai` - but that would maybe be less ideal because we'd need `cai.NewAsset`, `cai.AssetFromREST`, etc.

Reply:
will use caiasset, avoiding asset since it is a common variable name.